TokensExtraction.groovy

```diff
@@ -80,9 +80,9 @@ pipeline {
       steps {
         script {
           if (params.LIBRARIES.length() > 0) {
```

Comment:
I cannot make a comment on line 73, but can you change it for `npm install -g pnpm@7.0.0` ?